src/Resources/config/doctrine/model/GiftCardConfiguration.orm.xml

```diff
@@ -14,6 +14,7 @@
             </options>
         </field>
         <field name="defaultValidityPeriod" nullable="true" />
+        <field name="pdfRenderingCss" type="text" nullable="true" />
```

Comment:
What is this? Can I input CSS that will be applied to the gift card?

Reply:
Yes, exactly. No linter on this right now though :D